sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -533,20 +533,30 @@ object StringTrim {
 @ExpressionDescription(
   usage = """
     _FUNC_(str) - Removes the leading and trailing space characters from `str`.
-    _FUNC_(BOTH trimStr FROM str) - Remove the leading and trailing trimString from `str`
+    _FUNC_(BOTH trimStr FROM str) - Remove the leading and trailing `trimStr` characters from `str`
+    _FUNC_(LEADING trimStr FROM str) - Remove the leading `trimStr` characters from `str`
+    _FUNC_(TRAILING trimStr FROM str) - Remove the trailing `trimStr` characters from `str`
```

Comment:
I'd fold those lines into one if possible. If that looks quite messed, we could add each extra newline between `_FUNC_(...)`s. Currently, the doc is rendered as below:

![2017-09-21 3 33 09](https://user-images.githubusercontent.com/6477701/30693144-414283bc-9f09-11e7-86d0-52e2cfe83c4e.png)

Reply:
I see, thanks for finding this. I put the extra line there, then run this below
cd sql
create-docs.sh

it generate doc like this.

trim

trim(str) - Removes the leading and trailing space characters from str.

trim(BOTH trimStr FROM str) - Remove the leading and trailing trimStr characters from str

trim(LEADING trimStr FROM str) - Remove the leading trimStr characters from str

trim(TRAILING trimStr FROM str) - Remove the trailing trimStr characters from str